app/controllers/friendships_controller.rb

```diff
@@ -10,6 +10,8 @@ def update
     friendship = Friendship.find_by_id(fs.ids)
     friendship.confirmed = true
     friendship.save
+    reverse_fs = Friendship.new(user_id: current_user.id, friend_id: params[:id], confirmed: true)
+    reverse_fs.save
```

Comment:
- [ ] This save could be moved to the Friendship model using an after_create [callback ](https://guides.rubyonrails.org/active_record_callbacks.html#multiple-conditions-for-callbacks) so you don't do it within the controller